gender_novels/corpora/sample_novels/sample_novels.csv

```diff
@@ -87,3 +87,14 @@ author,date,title,country_publication,author_gender,filename,notes
 "Godwin, Francis",1768,"The Strange Voyage and Adventures of Domingo Gonsales, to the World in the Moon",England,male,godwin_voyage.txt,
 "Goldsmith, Oliver",1770,The Deserted Village,England,male,goldsmith_desert.txt,
 "Goulart, Ron",1960,Subject to Change,United States,male,goulart_subject.txt,
+"Eastman, Seth and Mary",1852,Aunt Phillis's Cabin,United States,both,eastman_phillis.txt,

```

Comment:
@equationcrunchor We hadn't though of "both" as a valid gender though of course it makes sense. During the meeting, can you make sure that someone takes on the task of a) integrating "both" as a valid input for Novel.author_gender and b) make sure that Corpus.count_authors_by_gender returns the proper counts, i.e. 'female' returns both 'female' and 'both'